File: myapp/child/ErrorHandler.cs
using sfn_ut.sfn;
using sfn_ut.sfn.error;

namespace sfn_ut.myapp.child;

public class ErrorHandler : SfnErrorHandler<ChildPayload> {
  public override SfnErrorHandlerInput<ChildPayload> Execute(SfnErrorHandlerInput<ChildPayload> payload) {
    return payload;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/child/ErrorHandler.java
package com.sfnut.myapp.child;

import com.sfnut.sfn.SfnErrorHandler;
import com.sfnut.sfn.error.SfnErrorHandlerInput;

public class ErrorHandler extends SfnErrorHandler<ChildPayload> {
    @Override
    public SfnErrorHandlerInput<ChildPayload> execute(SfnErrorHandlerInput<ChildPayload> payload) {
        return payload;
    }
}
